### ISSUE TITLE
Health equity calibration

### ISSUE THREAD

**Issue by cmhoove14** [cmhoove14/LEMMAABMv4#3]
Calibration underway for general transmission dynamics, but also want to calibrate to health disparities by race or at least geography.

Goal is to approximately match case and/or hospitalization rate(s) by racial/geographic group

Parameters to tweak could be race-based testing probabilities `Analysis/02-Prep-Par-Inputs ; input_pars$test_pars$race_test_mults`. Nothing else in model is explicitly tied to race, so might have to try deeper tweaks too

**Comment by cmhoove14** [comment]
First step is just to write scripts to assess outcomes produced by model simulations. `data/outputs/Test_Run_bta...` contains an .rds file with the simulation outputs:

- `epi_curve` contains number of agents in each step at each time step
- `infections` contains entries for every infection event
- `lineliest_tests` contains all tests conducted

So epi curve can be used for general calibration (e.g. to hospitalizations), but finer calibration by race and/or by geographies best done using `infections` and `linelist_tests`. Some ideas on how to calibrate/what we're looking for:

- High rates of positivity, but generally less tests conducted in census tracts (variable `ct`) with low healthy places index/high minority race/low income
- More infections in households with essential workers

**Comment by sblumberg** [comment]
Can we avoid race-based testing probabilities?  I think it would be a stronger analysis to identify if/when racial disparities arise organically via calibration to known data, rather than hard code racial inequity into simulation.

**Comment by cmhoove14** [comment]
@sblumberg Good point, and I agree. income already affects testing probability and now that synthetic population includes the healthy places index, have a few other variables that could be used (believe there's a healthcare access score, for instance)
